### ISSUE TITLE
CKV2_AWS_61 false positives for older Terraform S3 resource syntax

### ISSUE THREAD

**Issue by acdha** [bridgecrewio/checkov#4743]
If you have Terraform configuration using the older `aws_s3_bucket` syntax [which is supported until the AWS provider v5.0 is released](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/version-4-upgrade#changes-to-s3-bucket-drift-detection) you will have false positives reported for CKV2_AWS_61.

This is an example of the older syntax:

```hcl
resource "aws_s3_bucket" "example" {
  …
  lifecycle_rule {
    id      = "log-expiration"
    enabled = true
    prefix  = ""

    expiration {
      days = 180
    }
  }
}
```

Replacing this with the newer syntax will correctly detect the policy:

```hcl
resource "aws_s3_bucket_lifecycle_configuration" "logs" {
  bucket = aws_s3_bucket.example.id

  rule {
    id     = "ExpireAllObjects"
    status = "Enabled"

    expiration {
      days = 180
    }
  }
}
```

Ideally Checkov would support both forms because it's a non-trivial amount of work to rewrite all of your Terraform code and migrate the resources (or deal with some messy plans) and in either case the actual configured resource is identical.
